src/cloud-api-adaptor/test/e2e/common_suite.go

```diff
@@ -35,6 +35,18 @@ func DoTestCreateSimplePod(t *testing.T, e env.Environment, assert CloudAssert)
 	}
 }
 
+func DoTestLibvirtCreateSimplePodWithSecureCommsIsValid(t *testing.T, e env.Environment, assert CloudAssert) {
+	if !testCase_secureComms_isActive {
+		t.Skip("Skip - SecureComms is configured to be inactive - no need to test")
+	}
```

Comment:
We typically do the skip in the provider test case, so libvirt_test in this case, so it might be worth following this pattern.

Also our current pattern for whether to skip tests or not is to use environment variables, so can you use `TEST_E2E_SECURE_COMMS` rather than creating this extra go variable?

Reply:
I understand your reasoning, but none of the other providers would call this test, unless you are planning to support testing this elsewhere in future?

Reply:
I dont see why SecureComms will not be tested with other providers in the future.

Reply:
TEST_E2E_SECURE_COMMS is a matrix parameter. The right env is SECURE_COMMS. 

However I tried twice and the env variable is not set during the e2e test. Specifically i tried to replace `if !testCase_secureComms_isActive {` with `if os.Getenv("SECURE_COMMS") != "true" { ` and it failed. 
Before that I tried setting up an additional env variable and check if it exists in the test  - this also failed. Maybe this is because the env is different during the test. Anyhow I could not bring it to work the way you prefer it to be done using an env variable and therefore created the global variable instead.